docs/testing/types-of-testing.md

```diff
@@ -1 +0,0 @@
--- TO BE DEFINED --
```

Comment:
This has been moved to a section in the general document about automated testing: https://github.com/openeuropa/openeuropa/blob/OPENEUROPA-590/docs/automated-testing.md#types-of-tests